Source/URITemplate.swift

```diff
@@ -76,7 +76,13 @@ extension URITemplate: Equatable {
 
 extension URITemplate: Hashable {
     public var hashValue: Int {
-        return string.hashValue
+        var hasher = Hasher()
+        self.hash(into: &hasher)
+        return hasher.finalize()
+    }
```

Comment:
Do we need to provide `hashValue` now?
I expect that just implementing `hash(into:)` is enough.

Reply:
I left it for any unforseen compatibility issues, but the newer clients of Hashable will never call it anyway.

I will remove and push the change.